lib/bovine/views.py

```diff
@@ -7,20 +7,47 @@
 
 
 @api.route('/api/')
+def api_info():
+  return json.dumps({"hello": "world"}, indent=4)
```

Comment:
would be nice to respond with something useful... announcing that this service is the bovine-inventory api. 

let's maybe create a version string and respond with the version of the api too... we'd probably forget to bump the version though.  maybe tie it directly to the current git commit?